administrator/components/com_associations/views/associations/tmpl/default.php

```diff
@@ -53,7 +65,7 @@
 			</thead>
 			<tfoot>
 				<tr>
-					<td colspan="4">
+					<td colspan="<?php echo $colSpan; ?>">
```

Comment:
joao please read the code!

```php
$colSpan = $this->menuType == true ? 5 : 4;
```
so, colspan is 5 or 4 depending if we seleted menu item. ok correct

But ...
```php
if ($this->menuType == '')
{
	$colSpan--;
}
```
But if this var is empty we remove yet another column from the colspan?